brainframe/cli/dependencies.py

```diff
@@ -63,23 +63,6 @@ def _install_docker():
 
 docker = Dependency("docker", "install.ask-install-docker", _install_docker,)
 
-docker_compose = Dependency(
```

Comment:
If docker-compose is installed through your script right now, am I still able to call `docker-compos` commands via terminal, or is it only accessible through the `brainframe compose` command?

Reply:
It's only accessible through `brainframe compose`. Effectively, I'm treating Docker Compose like a Python library, and our version of Docker Compose doesn't get added to the user's path. If the user installs Docker Compose themselves, there's nothing stopping them from interacting with BrainFrame using their copy.

Reply:
Okay, that's fine by me. I just wanted to make sure that this installations docker-compose isn't available in the path anymore.